src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -250,20 +250,26 @@ NOTE: Future extensions are likely to add more types of sealed capabilities by e
 Unsealed capabilities::
 When `CT=0`, the capability authorizes access to a region of memory as defined by the permissions and bounds.
 
-[#sentry_cap,reftext="sentry capability"]
-Sentry capabilities::
+[#sealed_cap,reftext="sealed capability"]
+Sealed capabilities::
 Capabilities with `CT≠0` are sealed against modification.
 They cannot be dereferenced to access memory and it is not permitted to change the address, bounds or permissions.
 Modifications are restricted to _reducing_ the <<section_cap_level, CL>> field using <<ACPERM>>.
-+
+
+When a sealed capability is used as a function pointer it is known as a <<sentry_cap>>.
+
+[#sentry_cap,reftext="sentry capability"]
+==== Sentry Capabilities
 Sentries describe secure function entry points.
+They have a <<sec_cap_type>> of 1.
+
 They are used as the target of <<JALR_CHERI>>, which must have a zero offset for the use of a sentry to be valid.
-+
+
 Sentry capabilities can establish a form of control-flow integrity between mutually distrusting code.
 <<JALR_CHERI>> with zero offset automatically unseals a sentry target capability and installs it in the <<pcc,program counter capability>>.
-+
+
 <<JALR_CHERI>> also seals the return register, so the callee can return to the caller but cannot access the memory pointed at by the return register, and cannot return elsewhere.
```

Comment:
Hrm... On the one hand, this probably should say seals *as a sentry*... on the other hand, I'm happy it doesn't because Y-flavored CHERIoT won't do that.  And in fact CHERIoT's `cjalr` seals only if `cd` is `cra`.

Reply:
That's a good point, keeping it vague sounds good then